### PR TITLE
Fix: Remove unused functions

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] removes unused functions

### CODE REVIEW

File: manual/spam_challenge.php [hunk 1]
```diff
@@ -7,26 +7,10 @@ function plus($a, $b) {
...
 
-function gen_minus($a) {
-	return rand(0, $a);
-}
-
-function print_infix($name, $a, $b) {
-	return "$a $name $b";
-}
-
-function print_prefix($name, $a, $b) {
```
Comment (localheinz) [inline manual/spam_challenge.php]: Running

```shell
git grep -i have_search
```

on current `master` yields

```
include/site.inc:58:function have_search($site = FALSE)
```

File: bin/createNewsEntry [hunk 2]
```diff
@@ -160,20 +160,6 @@ function getContent(): string {
 	return $news;
 }
 
-function updateArchiveXML(string $id, string $archiveFile): void {
```
[the remark this answers is not in the document]
Comment (cmb69) [reply bin/createNewsEntry:163]: Yep, apparently a relict of 7971068c37a43fdd24f37542682729ab8bc6e7dd.

File: include/layout.inc [hunk 3]
```diff
@@ -59,12 +59,6 @@ function highlight_php_trimmed($code, $return = false)
     return null;
 }
 
-// Stats pages still need this
-function commonHeader($title) { site_header($title); }
```
[the remark this answers is not in the document]
Comment (cmb69) [reply include/layout.inc:63]: Not sure about this one; there is https://www.php.net/stats/, which appears to be (or have been generated) from mirrors?

File: include/layout.inc [hunk 4]
```diff
@@ -59,12 +59,6 @@ function highlight_php_trimmed($code, $return = false)
...
-
-
-// Stats pages still need this
-function commonFooter() { site_footer(); }
```
[the remark this answers is not in the document]
Comment (cmb69) [reply include/layout.inc:67]: Probably the same as with `commonHeader()`.

**Review by cmb69 — COMMENTED** [review]
I left some comments; not sure about the rest. I guess some help from somebody who knows about all the interactions, and which files are used/generated from where, would be useful.

**Review by kamil-tekiela — APPROVED** [review]
The `commonHeader` and `commonFooter` were probably used on mirrors running PHP 4 according to some Git history. I don't think they are used anywhere anymore. The stats page isn't used on main and I doubt that any private mirrors are relying on PHP's 20 year-old functions. 

Everything else seems to be unused. I can't find any way removing them would break the site.

**Comment by localheinz** [comment]
Rebased!

**Comment by localheinz** [comment]
Thank you, @cmb69 and @kamil-tekiela!

**Pull request merged**